lib/spack/spack/fetch_strategy.py

```diff
@@ -57,6 +57,15 @@
 from spack.util.compression import decompressor_for, extension
 
 import spack.util.pattern as pattern
+
+# The xmlrpclib module has been renamed to xmlrpc.client in Python 3
+# Spack does not yet support Python 3, but there's no reason not to be
+# future-proof
```

Comment:
Adding another external package seems like overkill for this. I'd rather just drop the Python 3 support.